docsets/discord.py/docset.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "discord.py",
```

Comment:
Please note that you need to add an archive at this path as well. I'll add it this time.